.editorconfig

```diff
@@ -9,7 +9,8 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.md]
-max_line_length = off
+# Lintian in Debian complains about lines > 512
+max_line_length = 512
```

Comment:
I think I'd prefer .md to be wrapping at 80 chars. This is what I am using everywhere I edit .md files -- it's sometimes a problem for URLs so we don't enforce that length.

Reply:
I agree, but I'm not sure what do you suggest to do here?

Reply:
We could set to 160 chars max length? That's less likely to cause trouble.

Reply:
Changed in #6378.